selectors.yml

```diff
@@ -135,7 +135,7 @@ selectors:
     name: "ethereum-mainnet-kroma-1"
   1088:
     selector: 8805746078405598895
-    name: "ethereum-mainnet-metis-1"
+    name: "ethereum-mainnet-andromeda-1"
```

Comment:
OK, this is just wrong; Everyone calls this network `Metis`, even https://l2fees.info/.
Calling it "Andromeda" is like renaming Mainnet to `ethereum-mainnet-homestead`.

chain-selectors is supposed to be the gate-keeper of proper naming convention. We shouldn't compromise here. RDD is internal, and can always rename later. And it should have the names registered here before pushing it elsewhere. We promise quick reviews.

Reply:
There was a situation with gnosis recently where we named it correctly in chain-selectors but in RDD it had been named "xdai mainnet" since we integrated with that chain a long time ago. Same situation with Metis.

Will make a note for eng-ops that the naming will be different for metis and see what the lift is to sync rdd with chain-selectors.